src/ztso.tex

```diff
@@ -20,7 +20,9 @@ \chapter{``Ztso'' Standard Extension for Total Store Ordering, v0.1}
 \begin{commentary}
   These rules render all PPO rules except \ref{ppo:fence}--\ref{ppo:rcsc} redundant.
   They also make redundant any non-I/O fences that do not have both PW and SR set.
-  Finally, they also imply that no instruction will be reordered past an AMO in either direction.
+  Finally, they also imply that no memory operation will be reordered past an AMO in either direction.
+  
+  In the context of RVTSO, as is the case for RVWMO, the storage ordering annotations are concisely and completely defined axiomatixally by PPO rules 5-7. In both of these memory models, it is the Load Value Axiom that allows a hart to forward a value from its store buffer to a programatically later load - that is to say that stores can be forwarded locally before they are visible to other harts.
```

Comment:
IIRC the rest of the document uses three hyphens and no spaces to represent an em dash.